app/routes/team-redirect.js

```diff
@@ -1,7 +1,7 @@
 import Route from '@ember/routing/route';
 
-export default Route.extend({
+export default class TeamRedirectRoute extends Route {
```

Comment:
Testing this route was initially confusing to me. According to `app/router.js`, we need to append `/team` to the base URL. If the redirect is successful, then the URL will have `/teams` at the end.

(I will create an application test in a separate pull request.)